composer.json

```diff
@@ -13,7 +13,8 @@
     "require-dev": {
         "php-vcr/phpunit-testlistener-vcr": "^1.1",
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "friendsofphp/php-cs-fixer": "~2"
```

Comment:
I agree that the 1 vs 2 will be an issue here but the rule of "anything that can be a phar should be" is not one I'm willing to compromise on. Many of the same dependencies used by these tools are also used by code generation libraries and one of those will probably end up in this SDK at some point. With PHP/composer only supporting a single version for each library I'd rather be extra careful and avoid version clashes.